static/src/js/components/OrderProgressList/__tests__/OrderProgressItem.test.js

```diff
@@ -1,57 +1,121 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
 
-import { Badge, ProgressBar } from 'react-bootstrap'
+import { render, screen } from '@testing-library/react'
+
 import { OrderProgressItem } from '../OrderProgressItem'
 
-import { retrievalStatusPropsEsi } from '../../OrderStatus/__tests__/mocks'
+import {
+  retrievalStatusPropsEsi,
+  retrievalStatusPropsSwotOrder,
```

Comment:
Should be named with `Swodlr`, not `Swot`, to match the other names that use the access method type